src/www/services_rfc2136_edit.php

```diff
@@ -58,6 +58,9 @@
     $pconfig['usetcp'] = isset($a_rfc2136[$id]['usetcp']);
     $pconfig['usepublicip'] = isset($a_rfc2136[$id]['usepublicip']);
 
+    $pconfig['recordtype'] = $a_rfc2136[$id]['recordtype'];
+    if (!$pconfig['recordtype']) $pconfig['recordtype'] = "both";
```

Comment:
yes. the config semantics are changed here so this imposes a silent migration only when the entry is edited. we cannot guarantee this won't be problematic in the future so we need to keep the empty value as default. :)